src/vs/workbench/browser/parts/editor/titleControl.ts

```diff
@@ -422,6 +422,14 @@ export abstract class TitleControl extends Themable {
 		return keybinding ? keybinding.getLabel() ?? undefined : undefined;
 	}
 
+	protected get tabHeight() {
+		return this.accessor.partOptions.tabHeight !== 'compact' ? 35 : 22;
```

Comment:
Maybe we put the constants into a similar `private static` thing as `TAB_HEIGHT` used to be before.